post_office/migrations/0008_attachment_headers.py

```diff
@@ -12,6 +12,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='attachment',
             name='headers',
-            field=models.JSONField(blank=True, null=True, verbose_name='Headers'),
+            field=models.TextField(blank=True, null=True, verbose_name='Headers'),
```

Comment:
Hey, sorry but is there a reason why we're changing past migrations?

Reply:
This approach doesn't feel right to me. How about adding a SQL migration which changes the type of that field by force.
The problem with backward compatibility is, that migration `0008_attachment_headers` already ran on the affected installations, so wouldn't have any effect anyway.

Reply:
Then we would need write custom SQL for each database that Django supports? :-/

Reply:
> This approach doesn't feel right to me. How about adding a SQL migration which changes the type of that field by force.

I agree with @jrief here.

How about this? We'll just add a new columns using Django's built in `JSONField` in this release and start reading from the new field, with fallback to the old one.

In a future version, we'll drop the old JSON field altogether.

Reply:
I think it is resolved by https://github.com/ui/django-post_office/pull/438 so I have removed all migrations from this PR.